src/Symfony/Component/JsonPath/Functions/CountFunction.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\JsonPath\Functions;
+
+/**
+ * @author Alexandre Daubois <alex.daubois@gmail.com>
+ */
+#[AsJsonPathFunction(name: 'count')]
+final class CountFunction extends AbstractJsonPathFunction implements JsonPathComparableFunctionInterface
```

Comment:
should these core functions be tagged as `@internal` as the JsonCrawler is registering them on its own and so there is no reason for ever accessing them directly outside the component ?